test/Swashbuckle.AspNetCore.SwaggerGen.Test/Fixtures/FakeControllerWithInheritDoc.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+
+namespace Swashbuckle.AspNetCore.SwaggerGen.Test;
+
+/// <inheritdoc cref = "FakeControllerWithXmlComments" />
+public class FakeControllerWithInheritDoc : FakeControllerWithXmlComments
+{
+    /// <inheritdoc cref = "FakeControllerWithXmlComments.ActionWithSummaryAndRemarksTags"/>
+    public new void ActionWithSummaryAndRemarksTags()
+    {
+        throw new NotImplementedException();
+    }
+
+    /// <inheritdoc cref = "FakeControllerWithXmlComments.ActionWithParamTags"/>
+    public new void ActionWithParamTags(string param1, string param2)
+    {
+        throw new NotImplementedException();
+    }
+
+    /// <inheritdoc cref = "FakeControllerWithXmlComments.ActionWithResponseTags"/>
```

Comment:
Only because I've never seen tags formatted this way before, so I think it just looks odd and out-of-place (I haven't seen if there's any already in this codebase 😄)

```suggestion
/// <inheritdoc cref="FakeControllerWithXmlComments" />
public class FakeControllerWithInheritDoc : FakeControllerWithXmlComments
{
    /// <inheritdoc cref="FakeControllerWithXmlComments.ActionWithSummaryAndRemarksTags"/>
    public new void ActionWithSummaryAndRemarksTags()
    {
        throw new NotImplementedException();
    }

    /// <inheritdoc cref="FakeControllerWithXmlComments.ActionWithParamTags"/>
    public new void ActionWithParamTags(string param1, string param2)
    {
        throw new NotImplementedException();
    }

    /// <inheritdoc cref="FakeControllerWithXmlComments.ActionWithResponseTags"/>
```